pkg/tests/end_to_end_tests/promql_endpoint_integration_test.go

```diff
@@ -275,7 +275,7 @@ func dateHeadersMatch(expected, actual []string) bool {
 		return false
 	}
 
-	return expectedDate.Sub(actualDate) <= 10*time.Second
+	return expectedDate.Sub(actualDate) <= 60*time.Second
```

Comment:
@niksajakovljevic should I leave this?

Reply:
I belive we can leave it. I think 10 seconds was picked arbitrarily to avoid flakiness